src/bp/ui-studio/src/web/translations/fr.json

```diff
@@ -250,7 +256,7 @@
         "executeAction": "Exécuter Action",
         "listen": "Écouter",
         "prompt": "Questionne",
-        "router": "Routeur",
+        "router": "Si Sinon",
```

Comment:
this really stinks + all node names are infinitive verbs so "Questionne" should be "Questionner"

Reply:
Ok but `Questionne` is not part of my PR changes. As for the If Else node the name is in the design https://app.zeplin.io/project/5e824d8b3e83946cb191a691/screen/5ed6ae0ade28f923f59a0fbd